silver/models/documents/base.py

```diff
@@ -34,32 +35,31 @@
 from django.db.models import Max, ForeignKey, F
 from django.template.loader import select_template
 from django.utils import timezone
+from django.utils.encoding import python_2_unicode_compatible
 from django.utils.text import slugify
 from django.utils.translation import ugettext_lazy as _
 from django.utils.module_loading import import_string
 
-from silver.models.billing_entities import Customer, Provider
 from silver.currencies import CurrencyConverter, RateNotFound
+from silver.models.billing_entities import Customer, Provider
+from silver.models.documents.entries import DocumentEntry
 from silver.models.documents.pdf import PDF
 from silver.utils.international import currencies
 
-from .entries import DocumentEntry
-
 
 _storage = getattr(settings, 'SILVER_DOCUMENT_STORAGE', None)
 if _storage:
     _storage_klass = import_string(_storage[0])
     _storage = _storage_klass(*_storage[1], **_storage[2])
 
-
 PAYMENT_DUE_DAYS = getattr(settings, 'SILVER_DEFAULT_DUE_DAYS', 5)
 
 logger = logging.getLogger(__name__)
 
 
 def documents_pdf_path(document, filename):
     path = '{prefix}{company}/{doc_name}/{date}/{filename}'.format(
-        company=slugify(unicode(
+        company=slugify(force_text(
```

Comment:
`force_text` is not imported in this file and will cause an error when this method is called.

Reply:
Good catch.